sqlglot/parser.py

```diff
@@ -6055,7 +6055,15 @@ def _parse_alter_table_alter(self) -> t.Optional[exp.Expression]:
             return self.expression(exp.AlterColumn, this=column, default=self._parse_conjunction())
         if self._match(TokenType.COMMENT):
             return self.expression(exp.AlterColumn, this=column, comment=self._parse_string())
-
+        if self._match_text_seq("DROP", "NOT" ,"NULL"):
+            return self.expression(exp.AlterColumn,
+                                   this=column,
+                                drop=True,
+                                   expression=exp.NotNullColumnConstraint)
```

Comment:
This needs to be `expression=exp.NotNullColumnConstraint()`, because it currently assigns a class to the arg.

Reply:
Rookie mistake, Thanks!